tests/DbaaS/verifyDBaaSPXCCluster_test.js

```diff
@@ -23,7 +23,7 @@ const singleNodeConfiguration = {
   memory: '1.2 GB',
   cpu: '0.2',
   disk: '25 GB',
-  dbType: 'MySQL 8.0.23',
+  dbType: 'MySQL 8.0.25',
```

Comment:
I would just create a variable, I know we added these in a rush, but you can just control this via one single variable